remix.init/index.js

```diff
@@ -39,24 +33,19 @@ async function copyTemplateFiles({ files, rootDirectory }) {
 
 async function updatePackageJsonForEdge(directory) {
   const packageJson = await PackageJson.load(directory);
-  const {
-    dependencies: { "@remix-run/node": _node, ...dependencies },
```

Comment:
Since both Edge & functions are removing `@remix-run/node`, I just removed it from `package.json`